npm/src/compiler/compiler.ts

```diff
@@ -37,24 +39,24 @@ export interface ICompiler {
 export type ICompilerWorker = ICompiler & IServiceProxy;
 export type CompilerState = ServiceState;
 
-function errToDiagnostic(err: any): VSDiagnostic {
-  if (
-    err &&
-    typeof err.severity === "string" &&
-    typeof err.message === "string"
-  ) {
-    err.start_pos = err.start_pos || 0;
-    err.end_pos = err.end_pos || 0;
-    return err;
-  } else {
-    return {
-      severity: "error",
-      message: err.toString(),
-      start_pos: 0,
-      end_pos: 0,
-    };
-  }
-}
+//function errToDiagnostic(err: any): VSDiagnostic {
+//  if (
+//    err &&
+//    typeof err.severity === "string" &&
+//    typeof err.message === "string"
+//  ) {
+//    err.start_pos = err.start_pos || 0;
+//    err.end_pos = err.end_pos || 0;
+//    return err;
+//  } else {
+//    return {
+//      severity: "error",
+//      message: err.toString(),
+//      start_pos: 0,
+//      end_pos: 0,
+//    };
+//  }
+//}
```

Comment:
Oops, forgot to remove this.

This was needed to handle events for katas because they were not being emitted in the same way that they were for non-kata code. I've changed it such that events are emitted the same way (in the wasm crate layer) so this code is no longer needed.

Reply:
Removed.